Styling_links readability (#24913)
mdn/content: files/en-us/learn/css/styling_text/styling_links/index.md

### Default styles

> The example below illustrates what a link will behave like by default (the CSS is enlarging and centering the text to make it stand out more). You can compare the behavior of the default stylings in the example with the behavior of other links on this page (which have some CSS stylings applied to them). Default links have the following properties:

Reviewer: ```suggestion
The example below illustrates what a link will look and behave like by default; though the CSS is enlarging and centering the text to make it stand out more. You can compare the look and behavior of the default stylings in the example with the look and behavior of other links on this page which have more CSS styles applied. Default links have the following properties:
```

- Active links are red. Try holding down the mouse button on the link as you click it.

```html
<p><a href="#">A simple link</a></p>
```

```css
p {
  font-size: 2rem;
  text-align: center;
}
```

{{ EmbedLiveSample('Default_styles', '100%', 130) }}

> > **Note:** All the links in the examples on this page are fake links — a `#` (hash/pound sign) is put in place of the real URL. This is because if the real links were included, clicking on them would break the examples (you'd end up with an error or a page loaded in the embedded example that you couldn't get back from). `#` just links to the current page.

Reviewer: ```suggestion
> **Note:** All the link examples on this page link to the top of their window. The empty fragment (`href="#"`) is used to create simple examples and ensure the live examples, which are each contained in an {{HTMLElement("iframe")}}, don't break.
```

The `#` is the equivalent of top of page. See https://web.dev/learn/html/links/

>Browsers also support two "top of page" links: clicking on <a href="#top">Top</a> (case-insensitive) or simply <a href="#">Top</a> will scroll the user to the top of the page, unless there is an element with the id of top set in the same letter casing.
